core-bundle/contao/classes/DataContainer.php

```diff
@@ -914,6 +914,18 @@ protected function generateGlobalButtons(/* DataContainerGlobalOperationsBuilder
 		$legacyCallback = function (DataContainerOperation $config) {
 			trigger_deprecation('contao/core-bundle', '5.6', 'Using a button_callback without DataContainerOperation object is deprecated and will no longer work in Contao 6.');
 
+			if (!\is_array($config['button_callback'] ?? null) && !\is_callable($config['button_callback'] ?? null))
+			{
+				return;
+			}
+
+			if ($config['icon'])
```

Comment:
```suggestion
			if ($config['icon'] ?? null)
```

Fixes the following error:

```
ErrorException:
Warning: Undefined array key "icon"

  at vendor\contao\contao\core-bundle\src\DataContainer\DataContainerOperation.php:78
  at Contao\CoreBundle\DataContainer\DataContainerOperation->offsetGet('icon')
     (vendor\contao\contao\core-bundle\contao\classes\DataContainer.php:922)
  at Contao\DataContainer->{closure:Contao\DataContainer::generateGlobalButtons():914}(object(DataContainerOperation))
     (vendor\contao\contao\core-bundle\src\DataContainer\AbstractDataContainerOperationsBuilder.php:195)
  at Contao\CoreBundle\DataContainer\AbstractDataContainerOperationsBuilder->executeButtonCallback(array('tl_files', 'syncFiles'), 
```